moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -68,10 +66,10 @@ export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
       type: "String",
     },
     project_description: {
-      hidden: true,
```

Comment:
I searched different places, and I cannot find where this `hidden` is being used or referenced. it was only on four columns, which makes me think its leftover from either GridTable or Material Table